metrotranscom/doorway
# feat: change input of rent search fields to be numeric, fix typo

# Pull Request Template

## Description

- Change the input type and add input mode and pattern props for the rent field. This is for both the landing and listings search filters.
- Fix an unrelated typo that made the "show matching listings" text not show properly.

## How Can This Be Tested/Reviewed?

- Go to the landing page. Try to enter letters in the max monthly rent box and you will see that these don't show. Try entering numbers and you will be able to.
- Try the same thing in the filters modal of the listings page.

Ideally we would test this on mobile and see that the numbers keyboard shows up rather than the text keyboard. I will test this once the changes are in dev.
